### PR TITLE
COMMERCE-12062, COMMERCE-12091

### PR DESCRIPTION
**Pull request by pietrobocale**
https://liferay.atlassian.net/browse/COMMERCE-12062
https://liferay.atlassian.net/browse/COMMERCE-12091

Base PR: https://github.com/liferay-commerce/liferay-portal/pull/4129

### CODE REVIEW

**Comment by pietrobocale** [comment]
ci:test:sf

**Comment by pietrobocale** [comment]
ci:stop

**Comment by pietrobocale** [comment]
ci:test:sf

**Comment by pietrobocale** [comment]
ci:test:commerce-functional

**Comment by pietrobocale** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceDiscounts.testcase [hunk 1]
```diff
@@ -924,6 +924,84 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-12062 and COMMERCE-12091, Verify that is not possible to use the comma and that the Minimum Amount must be filled with a number"
+	@priority = 5
+	test CanUserReciveAnErrorForEmptyOrDecimalNumberForMinimumAmountField {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceDiscounts.testcase:929]: Please update the test name according to the scenario covered

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceDiscounts.testcase [hunk 2]
```diff
@@ -924,6 +924,84 @@ definition {
...
+		task ("Given a Discount is created") {
+			CommerceJSONDiscountsAPI._addDiscount(
+				active = "true",
+				level = "L1",
+				percentageLevel1 = 10,
+				target = "products",
+				title = "Discount 1 10% off",
+				usePercentage = "true");
+
+			CommerceNavigator.gotoPortlet(
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceDiscounts.testcase:941]: Please use gotoDiscountsAdmin macro, thanks!

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by pietrobocale** [comment]
ci:test:sf

**Comment by pietrobocale** [comment]
ci:commerce-functional

**Comment by pietrobocale** [comment]
ci:test:commerce-functional

**Comment by pietrobocale** [comment]
ci:test:relevant

**Comment by lorenzolapescara** [comment]
Failures are not related to the PR.
@luca-pellizzon can you please force-forward this PR? Thanks!

**Comment by luca-pellizzon** [comment]
ci:forward:force

**Comment by luca-pellizzon** [comment]
ci:forward:force

**Pull request closed without merging**